components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/datasource/QueryExecutor.java

```diff
@@ -51,13 +54,17 @@ public boolean executeDeleteQuery(String uuid) throws SQLException, BusinessRule
         return statement.execute();
     }
 
-    public boolean executeUpdateBusinessRuleQuery(String uuid, Blob newBusinessRule, int deploymentStatus) throws SQLException, BusinessRulesDatasourceException {
+    public boolean executeUpdateBusinessRuleQuery(String uuid, byte[] newBusinessRule, int deploymentStatus) throws
+            SQLException, BusinessRulesDatasourceException {
         Connection conn = dataSource.getConnection();
-        PreparedStatement statement = getUpdateBusinessRuleQuery(conn, uuid, newBusinessRule, deploymentStatus);
+        Blob newBusinessRuleBlob = conn.createBlob();
+        newBusinessRuleBlob.setBytes(1,newBusinessRule);
```

Comment:
Check formatting. Also do the same for other places as well.

Reply:
Fixed by bd9e014